cqrs-example/cqrs-example.cabal

```diff
@@ -24,12 +24,12 @@ Executable cqrs-example
   Main-is:           CQRSExample/Main.hs
   Ghc-options:       -Wall
   Build-depends:     base >= 4.8 && < 5
-                   , aeson >= 0.8 && < 0.9
+                   , aeson >= 0.8
```

Comment:
Actually, just went ahead and did it, so you just need to rebase on top of master. (I've only compile-tested.)